### DISCUSSION TITLE
Flareact future

### DISCUSSION THREAD

**Discussion by Giggiux** [flareact/flareact#144] [Q&A] [answered]
I am considering using Flareact on a small side project, but I have a question regarding the future of the framework:

is this gonna be maintained constantly and updated with new features, as Next.js is, or it's not going to be -at least in the short term- something good for production sites? 

In general, what are the plan for it?

I love the idea behind Flareact, because I'm fascinated by Cloudflare Workers, and I've found Next.js to be very simple to use, so something that "recalls" the same kind of APIs, but on Workers is appealing to me :D 

Thanks!

**Comment by traffisco** [comment]
Future is not clear....
For a small side project, you'll be fine. Maybe you'll have to fork and apply some patches.
I have flareact running in production for couple of projects with few selected unmerged PRs without any issue

**Comment by jplhomer** [comment, marked as answer]
Hi @Giggiux! There are no plans for immediate work on Flareact. This started as an experiment I built in a week in 2020, and iterated on a little bit since.

Since I don't have time to work on this project, nor am I using it anywhere for personal or professional projects, I don't have plans to contribute to it at this time. However, @tj-kev has offered to step in as a maintainer for the time being, and depending on his availability, may be able to keep pushing the project forward ❤️ 

There are also a few factors to consider here:

- Since Flareact was released, [Cloudflare Pages](https://pages.cloudflare.com/) was announced and made generally available. This is Cloudflare's opinionated version of shipping JAMStack sites. While they don't offer a tightly-integrated dynamic/static combination _yet_, they have been talking about adding first-class support for shipping a static site along with API functions in an opinionated way. This feels like it might be a nice replacement for Flareact, and it's backed by Cloudflare itself.
- Cloudflare also announced their [intentions to support native Node.js APIs in Workers](https://blog.cloudflare.com/node-js-support-cloudflare-workers/). While it won't happen right away, my guess is that in the next 6-12 months, we'll live in a world where Next.js is supported on Workers 🎉 This has been the main selling point of Flareact — so Next.js support would negate any real benefit of using Flareact for a long-term project.

Feel free to use it for side projects and experiments 💪 have fun!

**Reply by Giggiux** [reply]
Thanks a lot for the perfect explanation! :D

**Reply by jplhomer** [reply]
18 months later 🎉 https://blog.cloudflare.com/next-on-pages/
